tests/snapshots.test.js

```diff
@@ -3,17 +3,33 @@ const snapshotsTest = require('../snapshots');
 
 const PORT = process.env.PORT || 8101;
 
+/**
+ * Combined examples that embed responsive examples.
+ * @type {Map<string, boolean>} Mapping of example urls to whether they embed responsive examples.
+ */
+const RESPONSIVE_COMBINED_EXAMPLES = new Map([
```

Comment:
Is there a particular reason for this being a Map instead of simple array?

Reply:
This is a habit of mine when I have a specific value that I'm searching for, since it's faster, but I suppose in the case of a two-value array it wouldn't really matter